src/Modules/Reputation/interfaces/ReputationTokenInterface.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+interface ReputationTokenInterface {
+    struct ReputationAccountInfo {
+        uint256 locked;
+        uint256 lastDecayEpoch;
+        uint256 frozenUntilEpoch;
+    }
+
+    event ReputationFrozen (
```

Comment:
Events do not belong in the interface as they are never needed to use an interface and on-chain conformity of event declaration cannot be achieved with just an interface. Please move events into the processing contract of this PR.